chapters/ch56.md

```diff
@@ -30,7 +30,7 @@ Chúng ta hãy quay trở lại ứng dụng xe tự lái, trong đó thuật to
 
 > 3. How far is the overall system’s performance from human-level performance? Here, human-level performance assumes the human has to plan a path for the car given only the outputs from the previous two pipeline components (rather than access to the camera images). In other words, how does the Plan path component’s performance compare to that of a human’s, when the human is given only the same input?
 
-3. Cách biệt của toàn hệ thống và con người? Ở đây, chất lượng mức con người giả định rằng con người phải hoạch định một con đường cho chiếc xe từ đầu ra từ hai pipeline trước đó (không phải truy cập vào hình ảnh). Nói cách khác, so sánh hiệu suất của phần pipeline hoạch định đường đi so với con người, khi cả hai được cung cấp cùng một đầu vào?
+3. Cách biệt của toàn hệ thống và con người? Ở đây, con người phải hoạch định một con đường cho chiếc xe từ đầu ra của hai pipeline trước đó (không phải truy cập vào hình ảnh). Nói cách khác, so sánh hiệu suất của phần pipeline hoạch định đường đi so với con người, khi cả hai được cung cấp cùng một đầu vào?
```

Comment:
3. Cách biệt giữa khả năng của toàn hệ thống và con người tới cỡ nào? Đối với con người, chúng ta giả định rằng con người  tính đường đi cho xe mà chỉ dựa vào kết quả đầu ra của hai pipeline trước đó (thay vi dựa vào hình ảnh camera). Nói cách khác, với cùng thông tin đầu, khả năng ước lượng đường đi của thuật toán so với con người  sẽ như thế nào?